expression/builtin_string.go

```diff
@@ -3491,5 +3493,67 @@ type loadFileFunctionClass struct {
 }
 
 func (c *loadFileFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
-	return nil, errFunctionNotExists.GenWithStackByArgs("FUNCTION", "load_file")
+	if err := c.verifyArgs(args); err != nil {
+		return nil, err
+	}
+	valStr, _ := ctx.GetSessionVars().GetSystemVar(variable.MaxAllowedPacket)
+	maxAllowedPacket, err := strconv.ParseInt(valStr, 10, 64)
+	if err != nil {
+		return nil, err
+	}
+	bf := newBaseBuiltinFuncWithTp(ctx, args, types.ETString, types.ETString)
+	sig := &builtinInstrLoadFileSig{bf, maxAllowedPacket}
+	return sig, nil
+}
+
+type builtinInstrLoadFileSig struct {
+	baseBuiltinFunc
+	maxAllowedPacket int64
+}
+
+func (b *builtinInstrLoadFileSig) Clone() builtinFunc {
+	newSig := &builtinInstrLoadFileSig{}
+	newSig.cloneFrom(&b.baseBuiltinFunc)
+	newSig.maxAllowedPacket = b.maxAllowedPacket
+	return newSig
+}
+
+//MySQL implementation return NULL value when error occurs
```

Comment:
It is better to add a space at the start of the comment.

Reply:
The comment for this function should begin with `evalString` ...
https://golang.org/doc/effective_go.html#commentary

> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

Reply:
@cfzjywxk PTAL at this comment.